packages/js/src/initializers/elementor-editor-integration.js

```diff
@@ -91,6 +91,43 @@ function initializePostStatusListener() {
 	} );
 }
 
+/**
+ * Checks if field is keyword field and detect if it is not changed.
+ *
+ * @param {HTMLElement} input The input.
+ *
+ * @returns {boolean} true if input is keyword field and input by user value not changed.
+ */
+function isKeywordFieldUnchanged( input ) {
```

Comment:
Minor improvement: You could choose to split this function up into two:
* One function that checks if this input field is a keyword field 
    * gets one argument: the input field name.
* Another one that checks if the input field is unchanged.
    * gets two arguments: the old value and the new value. 

It would separate these two concerns and reduce some code coupling. Although I may be nitpicking in this case.

Reply:
Then we can use it like this, which I think is still readable:
```js
if ( isKeywordField( input.name ) && isUnchanged( input.oldValue, input.value ) ) {
     return;
}
```
What do you think?